fuzz/README.md

```diff
@@ -0,0 +1,10 @@
+
+# Fuzzing i686 on native `x86_64` linux
```

Comment:
The codification of i686 vs `x86_64` is inconsistent here (also linux might be better off as Linux)

Reply:
It's consistent with the names in `rustup target list`. I'll change the capital of linux, good point.

Reply:
I mostly meant the code blocks around `x86_64` and lack thereof around i686

Reply:
Ohhhh. That makes sense then xd

Reply:
Just checked that Github's markdown flavor can render inline code in titles so I'll add it to both. I vaguely remember that jekyll (the Github pages app) could not.